src/lib/authentication/services/AuthenticationService.js

```diff
@@ -20,7 +20,12 @@ class AuthenticationService {
 
   logout = () => {
     const logoutUrl = `${invenioConfig.APP.REST_ENDOINTS_BASE_URL}/logout`;
-    return http.post(logoutUrl);
+    if (invenioConfig.APP.LOGOUT_METHOD === 'GET') {
+      window.location = logoutUrl;
```

Comment:
I wouldn't assign a string to `window.location` which is an object. We can use the `replace` method.
```suggestion
      window.location.replace(logoutUrl);
```